src/containerapp/azext_containerapp/custom.py

```diff
@@ -577,7 +586,8 @@ def update_containerapp(cmd,
                         workload_profile_name=None,
                         termination_grace_period=None,
                         no_wait=False,
-                        secret_volume_mount=None):
+                        secret_volume_mount=None,
+                        source=None):
```

Comment:
In this PR, you also want to support --source for `update` command?

Reply:
yes, we are adding support for `--source` as well for `update` in this PR.